R/parse.R

```diff
@@ -9,11 +9,12 @@
 #'   * A column "pos_id" for (positional id) which can be used for sorting
 #'     (because "id" cannot be used in general). Note that the nth value of this
 #'     column corresponds to n as long as no tokens are inserted.
-#'   * A column "child" that contains the nested subtibbles.
+#'   * A column "child" that contains the nested sub-tibbles.
```

Comment:
Do we want to use the "nest" terminology here (maybe later)?

Reply:
Good idea.